e2e-test/application_sync_test.go

```diff
@@ -114,9 +114,7 @@ var _ = Describe("Test the application synchronizing", func() {
 			Expect(lrr.Records[1].Name).Should(Equal("test-v2"))
 
 			// The workflow includes a suspend step.
-			if lrr.Records[1].Status != "suspending" {
-				return fmt.Errorf("the record status is %s, not suspending", lrr.Records[1].Status)
-			}
+			Expect(lrr.Records[1].Status).Should(Equal("suspending"))
```

Comment:
Does this change mean that can't wait for the status become to suspending?

Reply:
No it stays the same with the old one.